.github/workflows/build_documentation.yml

```diff
@@ -114,34 +114,6 @@ jobs:
       working-directory: ./doc/build
       run: cmake --build . -j 4
 
-    - name: Upload Acknowledgments
```

Comment:
Removed these 4 from the GitHub Action artifact uploads since they are no longer produced as PDF.